lib/src/rules/avoid_redundant_argument_values.dart

```diff
@@ -63,14 +63,18 @@ class _Visitor extends SimpleAstVisitor {
   _Visitor(this.rule, this.context);
 
   void check(ArgumentList argumentList, List<ParameterElement> parameters) {
-    if (argumentList.arguments.isEmpty || parameters == null) {
+    final arguments = argumentList.arguments;
+    if (arguments.isEmpty || parameters == null) {
```

Comment:
This code no longer references the parameter `parameters`. Consider removing it.

Reply:
Done.